kafka_roller/kafka_roller/tasks.py

```diff
@@ -103,11 +105,14 @@ def stop(c, pre_stop=None):
     # introduce & execute post stop here if needed..
 
 
-def start(c, pre_start=None):
+def start(c, pre_start=None, start_cmd=None):
     """Gracefully start an instance."""
     if pre_start is not None:
         _run_safe(c, "Pre-start", pre_start)
-    service_start(c, "kafka")
+    if start_cmd is not None:
+        _run_safe(c, "Start", start_cmd)
```

Comment:
Do we still need to start the service in this case?

Reply:
The intention is to use customized start-cmd as the service start command.
Therefore, we shouldn't need to call the service_start function again.

In the scenario of a server reboot, passing in a "reboot" would serve the
purpose of service restart, as the services should be automatically brought
up at boot up.
In the scenario of VM reinitialization, we would just reinitialize the VM,
and the automation that initializes the VM should start the service in
fresh.